demos/question-answering/README.md

```diff
@@ -23,44 +38,83 @@ To complete the tutorial follow the steps below:
+> - Vector Store: `dpoulopoulos/qna-vectorstore:46a578c`
+> - LLM Predictor: `dpoulopoulos/qna-predictor:a6e08d4`
+> - LLM Transformer: `dpoulopoulos/qna-transformer:35e3990`
 
-Once the images are ready, proceed to run the Notebooks. The project consists of four Notebooks. Launch and run each Notebook to explore and execute the experiment end-to-end:
+Once the images are ready, proceed to run the Notebooks. The project consists of four Notebooks. Launch and run each
+Notebook to explore and execute the experiment end-to-end:
 
-1. `01.create-vectorstore`: Load the documents from your private corpus (e.g., the `documents` folder), process them, and create the Vector Store.
+1. `01.create-vectorstore`: Load the documents from your private corpus (e.g., the `documents` folder), process them,
+    and create the Vector Store.
 1. `02.serve-vectorstore`: Create an inference service for the Vector Store.
 1. `03.document-precition`: (optional): Invoke the Vector Store inference service.
 1. `04.serve-llm`: Create an inference service for the LLM.
-1. `05.question-answering`: Invoke the LLM. Post a question to the LLM inference service and get back a human-like answer.
+1. `05.question-answering`: Invoke the LLM. Post a question to the LLM inference service and get back a human-like
+   answer.
 
 The last Notebook outlines the user's perspective. The application flow is depicted in the following figure:
 
-![flow-chart](images/LLM-flowchart.svg)
-
-1. User: Create a new request to the LLM inference service with a query.
-1. LLM Transformer: Intercept the request, extract the user's query, and create a new request to the Vector Store inference service passing the user's question in the payload.
-1. Vector Store: Accept the request from the LLM Transformer extract the user's question and based on the user's question retrieve the `4` most relevant documents.
-1. Vector Store: Respond to the LLM Transformer.
-1. LLM Transformer: Get the `4` most relevant documents from the Vector Store response.
-1. LLM Transformer: Create a new request to the LLM Predictor passing the `4` most relevant documents as context and the user's question.
-1. LLM Predictor: Accept the request from the LLM Transformer, extract the user's question as well as the context, and answer the user's question based on the relevant context.
-1. LLM Predictor: Respond to the LLM Transformer with the completion prediction.
\ No newline at end of file
+![flow-chart](images/LLM-flowchart.png)
+
+1. User: Transform raw documents into sentence embeddings.
+1. User: Ingest document embeddings, documents data, and metadata in the Vector Store.
+1. User: Ask a new question.
+1. LLM Transformer: Intercept the request, extract the user's query, and create a new request to the Vector Store
+   Inference Service passing the user's question in the payload.
+1. Vector Store Predictor: Accept the request from the LLM Transformer extract the user's question and ask the Vector
```

Comment:
"Extract the user's question from the request of the LLM Transformer and ask..."